Presentation.Web/app/shared/datePicker/datePicker.directive.ts

```diff
@@ -0,0 +1,23 @@
+﻿
+((ng, app) => {
+    'use strict';
+
+    app.directive("datePicker", [
+        () => ({
+            templateUrl: "app/shared/datePicker/datePicker.view.html",
+            scope: {
+                id: "@",
+                model: "=ngModel",
+                appendurl: "@",
+                disabled: "=ngDisabled",
+                required: "@"
+            },
+            link: ($scope) => {
```

Comment:
https://stackoverflow.com/questions/41160798/which-is-preferable-controller-or-link-function-in-angularjs-directives

Reply:
👍